test/execute.js

```diff
@@ -441,7 +441,7 @@ describe('execute', () => {
       })
     })
 
-    it('should handle requestContentType', function () {
+    it('should not add content-type with no form-data or body param', function () {
```

Comment:
you're right @bodnia it does _only_ make sense if there is a body. The user can override the `content-type` header in other ways, if need be IIRC.